frontend/packages/api_client/src/models/DisplayStaticContentPages.ts

```diff
@@ -54,7 +54,7 @@ export interface DisplayStaticContentPages {
      * @type {boolean}
      * @memberof DisplayStaticContentPages
      */
-    privacyPolicy: boolean;
+    privacy: boolean;
```

Comment:
@0x29a, how is this and the other changes in this file related to the Koa upgrade? 🤔

Reply:
@lgp171188, we don't modify this file manually. I just ran [update-api-client.sh](https://github.com/open-craft/opencraft/blob/master/frontend/scripts/update-api-client.sh) script to reflect changes in the `help_text` (it fetches API schema using Swagger endpoing). Looks like someone has forgotten to do this after changing backend code.